kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/GennyService.java

```diff
@@ -2,46 +2,26 @@
 
 import life.genny.qwandaq.EEntityStatus;
 import life.genny.qwandaq.entity.BaseEntity;
-import life.genny.qwandaq.models.ServiceToken;
-import life.genny.qwandaq.models.UserToken;
 import life.genny.qwandaq.utils.*;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.jboss.logging.Logger;
 
 import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
 import javax.json.bind.Jsonb;
 import javax.json.bind.JsonbBuilder;
 import java.lang.invoke.MethodHandles;
 
 @ApplicationScoped
-public class GennyService {
+public class GennyService extends KogitoService {
 
 	static final Logger log = Logger.getLogger(MethodHandles.lookup().lookupClass());
 
 	static Jsonb jsonb = JsonbBuilder.create();
 
+	// TODO: We need to get rid of this. This breaks multitenancy. Should not be decided once on startup.
```

Comment:
I think we should just remove this file all together

Reply:
Agree with Jasper. We could do it later, after thorough research.